pkg/cli/setup/application.go

```diff
@@ -76,6 +88,17 @@ func ScaffoldApplication(directory string, name string) error {
 		return err
 	}
 
+	bicepConfigFilepath := filepath.Join(directory, "bicepconfig.json")
+	_, err = os.Stat(bicepConfigFilepath)
+	if os.IsNotExist(err) {
+		err = os.WriteFile(bicepConfigFilepath, []byte(bicepConfigTemplate), 0644)
+		if err != nil {
+			return err
+		}
+	} else if err != nil {
+		return err
+	}
+
```

Comment:
Does the interactive display from `rad init` include this info in its output?

Reply:
It should. The Run command would return an error if scaffolding fails. Tested forcing an error to confirm that it does (this may not be the actual error, this was just a dummy message).
 
![Screenshot 2024-06-06 at 1 45 10 PM](https://github.com/radius-project/radius/assets/42750942/3e719af0-106e-424d-bc11-88f6d884f02c)